<launch>
    <rosparam file="$(find perception_pipeline)/config/lidar-config.yaml" />

    <node name="sync_node" pkg="perception_pipeline" type="sync_node" output="screen"></node>

    <node name="perception_node" pkg="perception_pipeline" type="perception_node" output="screen"></node>

    <node name="static_transforms" pkg="perception_pipeline" type="static_transforms" output="screen">
        <rosparam file="$(find perception_pipeline)/config/transforms.yaml" />
    </node>

</launch>
